config/201-clusterrole.yaml

```diff
@@ -26,15 +26,15 @@ rules:
   - cronjobsources
   - githubsources
   - kuberneteseventsources
-  verbs:
+  verbs: &everything
```

Comment:
I'm less of a fan of this style of YAML object references, because if we remove or change an apiGroup, it can possibly trigger changes through the whole file by accident.

Reply:
Personally, I think this is an improvement, but if you don't, I can remove the RBAC parts of this PR.